web/src/components/tasks/business-formation/NexusFormation.test.tsx

```diff
@@ -390,52 +389,5 @@ describe("<NexusFormationFlow />", () => {
         });
       });
     });
-
-    describe("when feature flag is not set", () => {
```

Comment:
Same comment, I think this test needs to stay in place because it describes the correct behavior for the FLP case